packages/tanstack-react-query/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@trpc/tanstack-react-query",
```

Comment:
Should we name the package differently? `@trpc/react-query` and `@trpc/tanstack-react-query` is potentially confusing but I'm not sure what's better really

`@trpc/tanstack-react-query-options` maybe?

Or maybe this client needs adding to the existing client package?

It's probably fine as is but it's worth questioning